modules/exploits/unix/http/zivif_ipcheck_exec.rb

```diff
@@ -0,0 +1,88 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'             => "Zivif Camera Web Interface Blind Remote Command Execution ",
```

Comment:
Usually the module describes the vulnerable functionality to prevent collisions if more than one bug are found.

Suggest:

```suggestion
      'Name'             => 'Zivif Camera iptest.cgi Blind Remote Command Execution',
```